### DISCUSSION TITLE
Render output: export ranges of frames

### DISCUSSION THREAD

**Discussion by ErwinBr** [opentoonz/opentoonz#6535] [Ideas]
### Is your feature request related to a problem? Please describe

The Sprite Sheet rendering output setting is super useful due to it's cropping/trimming functionality and how it's organizing the sheet vertically/horizontally/ or in a grid while keeping the images offset in the right position. The range of exported frames dictates a minimum witdh and height of the frames in order to keep the offset right. Therefore, it's a key functionality for exporting out sprites for video games, my main usecase for OpenToonz. In order to render a useful spritesheet, one needs to organize all frames to be in one single range of frames. This can be a pain to manage.

What would make the Sprite Sheet setting even more powerful for this specific usecase is the ability to export ranges of frames into the Sprite Sheet, allowing one to skip duplicate frames that you often have in an animation.

### Describe the solution you'd like

The way I envision it is comparable to the options Microsoft gives you when printing pages of a document. Using "," and "-" symbols, like so:

1, 2, 5 : Render frame 1, 2, and 5.
1 - 5 : Render frame 1 to 5
1-3, 6 : Render frame 1 to 3, and 6
1-3, 5-6 : Render frame 1 to 3, and 5 to 6

I realize that this way of rendering might not be valuable for all sorts of usecases, so it could for example be optional or linked specifically to Sprite Sheet rendering mode.

**Comment by RodneyBaker** [comment]
As a follow up to this request to be able to export/render specific frames or ranges of frames there is currently one way to do this although it is not immediately obvious.

The short of it is that we can export/render previewed frames.
As such, if we have preview on and only preview specific frames we can then Right Click and Save Previewed Frames.
